src/app/authoring-tool/edit-dynamic-prompt/edit-dynamic-prompt.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { DynamicPrompt } from '../../../assets/wise5/directives/dynamic-prompt/DynamicPrompt';
+import { ProjectService } from '../../../assets/wise5/services/projectService';
+
+@Component({
+  selector: 'edit-dynamic-prompt',
+  templateUrl: './edit-dynamic-prompt.component.html',
+  styleUrls: ['./edit-dynamic-prompt.component.scss']
+})
+export class EditDynamicPromptComponent implements OnInit {
+  allowedConnectedComponentTypes: string[] = ['OpenResponse'];
+  @Input() authoringComponentContent: any;
+  @Output() dynamicPromptChangedEvent = new EventEmitter<string>();
```

Comment:
We don't do anything with the string, so ok to emit void. Right now, in toggleDynamicPrompt(), we're actually emitting MatCheckboxChange event, and not doing anything with it.
```suggestion
  @Output() dynamicPromptChangedEvent = new EventEmitter<>();
```